### PR TITLE
Task: search for movies by keywords

### PR DESCRIPTION
**Pull request by sergeiown**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,29 +1,181 @@
...
 
-const textDisplay = document.getElementById("text-display");
-const textEditor = document.getElementById("text-editor");
+/* waiting for user actions with the form */
+searchForm.addEventListener("submit", async (event) => {
+  event.preventDefault();
 
-window.addEventListener("keydown", actionOnEvent);
+  paginationContainer.style.display =
+    "none"; /* prevent pagination with no search result */
```
Comment (UniBreakfast) [inline js/app.js:13]: It's generally considered better to hide/show with adding/removing classes than with direct inline style changes.

File: js/app.js [hunk 2]
```diff
@@ -1,29 +1,181 @@
...
 
-let isEditing = false;
+  const title = document.querySelector("#title").value;
+  const type = document.querySelector("#type").value;
```
Comment (UniBreakfast) [inline js/app.js:16]: What's the point in doing this?

Comment (UniBreakfast) [reply js/app.js:16]: You're just storing initial values in constants.

File: js/app.js [hunk 3]
```diff
@@ -1,29 +1,181 @@
...
+}
+
+/* displaying number of pages and controls */
+function displayPagination(totalResults, title, type, page) {
+  let currentPage = page;
+  let totalPages = Math.ceil(totalResults / 10);
+
+  if (totalPages > 100) {
+    totalPages = 100;
+  }
```
Comment (UniBreakfast) [inline js/app.js:51]: `Math.min(100, Math.ceil(totalResults / 10))`

File: js/app.js [hunk 4]
```diff
@@ -1,29 +1,181 @@
...
+  console.log(`totalPages: ${totalPages}`);
+}
+
+/* displaying movie data and buttons for details */
+function displayResults(movies) {
+  resultsContainer.innerHTML = "";
+
+  if (movies.length === 0) {
+    resultsContainer.innerHTML = `
+        <h1>Movie not found!</h1>`;
```
Comment (UniBreakfast) [inline js/app.js:107]: why two lines of code?

File: js/app.js [hunk 5]
```diff
@@ -1,29 +1,181 @@
...
+
+  for (const movie of movies) {
+    const div = document.createElement("div");
+
+    let realPoster;
+    if (movie.Poster === "N/A") {
+      realPoster = "./img/no_image.jpg";
+    } else {
+      realPoster = movie.Poster;
     }
```
Comment (UniBreakfast) [inline js/app.js:119]: a good place for ternary

**Review by UniBreakfast — APPROVED** [review]

File: js/app.js (hunk 3 above)
Comment (sergeiown) [reply js/app.js:51]: Thank you! Done!

File: js/app.js (hunk 5 above)
Comment (sergeiown) [reply js/app.js:119]: realPoster =
    movie.Poster === "N/A"
      ? (realPoster = "./img/no_image.jpg")
      : (realPoster = movie.Poster);

File: js/app.js (hunk 1 above)
Comment (sergeiown) [reply js/app.js:13]: My little rebellion )

File: js/app.js (hunk 2 above)
Comment (sergeiown) [reply js/app.js:17]: Resolved by agreement of the parties ))

**Pull request merged**